themes/bootstrap3/js/covers.js

```diff
@@ -2,22 +2,44 @@
 function loadCoverByElement(data, element) {
   var url = VuFind.path + '/AJAX/JSON?method=' + 'getRecordCover';
   var img = element.find('img');
-  var spinner = element.children('div.spinner');
-  var container = element.children('div.cover-container');
+  var spinner = element.find('div.spinner');
+  var container = element.find('div.cover-container');
+  var source = $('<p class="cover-source">' + VuFind.translate('cover_source_label') + ' </p>');
   function coverCallback(response) {
-    spinner.hide();
-    container.show();
     if (typeof response.data.url !== 'undefined' && response.data.url !== false) {
       img.attr("src", response.data.url);
-      container.children().not("img").hide();
+      var inlink = element.parent().is('a');
```

Comment:
Can we add a class to the expected link? This seems easy to trip with customizations. I can come back to this in the future if it proves to be especially tricky.

Reply:
@crhallberg How do you mean this? Do you think we should add a class into html template and use it then in covers.js for identification to be more specific? Or do you think add a class by this javascript code to be able to select it using CSS?

Reply:
@crhallberg can correct me if I'm wrong, but I suspect he's suggesting the former: adding a class into the HTML template to reduce the odds of this code hooking the wrong element because of a fairly broad selector.

Reply:
Thanks, I added class to templates

Reply:
Thanks! @crhallberg, are you happy with the `record-link` class, or would you prefer `cover-link` or something else? (I'm happy to make the change if necessary -- not trying to drive @xmorave2 crazy with nitpicky changes :-) ).

Reply:
I think `cover-link` or `record-cover-link` would be more appropriate.

Reply:
Thanks, @crhallberg! Do you have a preference, @xmorave2? I hope to do final testing and merging of this PR next week, so I can adjust the class name at that time if you have a preference.

Reply:
I choose record-cover-link. I pushed a commit with this change